packages/cosmic-swingset/lib/ag-solo/init-basedir.js

```diff
@@ -102,23 +102,38 @@ export default function initBasedir(
       fs.mkdirSync(agchServerDir);
       // we assume 'ag-cosmos-helper' is on $PATH for now, see chain-cosmos-sdk.js
       const keyName = 'ag-solo';
-      const password = 'mmmmmmmm\n';
       // we suppress stderr because it displays the mnemonic phrase, but
       // unfortunately that means errors are harder to diagnose
       execFileSync(
         'ag-cosmos-helper',
-        ['keys', 'add', keyName, '--home', agchServerDir],
+        [
+          'keys',
+          'add',
+          '--keyring-backend=test',
+          keyName,
+          '--home',
+          agchServerDir,
+        ],
         {
-          input: Buffer.from(password),
+          input: Buffer.from(''),
           stdio: ['pipe', 'ignore', 'ignore'],
         },
       );
       console.log('key generated, now extracting address');
       const kout = execFileSync(
         'ag-cosmos-helper',
-        ['keys', 'show', keyName, '--address', '--home', agchServerDir],
+        [
+          'keys',
+          'show',
+          '--keyring-backend=test',
```

Comment:
Is this password-less backend going to stick around? If it's only for testing, do we need to worry about it getting removed?

Reply:
It will stick around, or there will be a different password-less backend (since the use case is necessary for things like the IBC relayer, as discussed on Slack).  So yes, this is useful, and won't paint us into a corner.